source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -110,442 +110,528 @@ volume and a supporting
+           All storage units are in SI values, e.g. Gi = GiB = 1024\ :sup:`3` bytes.
+     
+           The total storage capacity for the tenant to have across all volumes.
+           The value should be an integer and a standard storage abbreviation, such as ``Gi`` or ``Ti``.
+
+           For example, 16 Ti for 16 Tibibytes.
+
+         - :mc:`~kubectl minio tenant create --servers`
 
-The remaining services support Tenant operations and are not intended for 
-consumption by users or administrators.
+           The total number of MinIO server pods to deploy in the Tenant.
+    
+           The Operator by default uses pod anti-affinity, such that the Kubernetes cluster *must* have at least one worker node per MinIO server pod.
+
+         - :mc:`~kubectl minio tenant create --volumes`
+
+           The total number of storage volumes (Persistent Volume Claims). 
+           The Operator generates an equal number of PVC *plus two* for supporting Tenant services (Metrics and Log Search).
+       
+           The specified :guilabel:`Storage Class` *must* correspond to a set of Persistent Volumes sufficient in number to match each generated PVC.
+    
+           The total number of persistent volume claims (``PVC``) per server is determined by dividing the number of volumes by the number of servers.
+           The storage available for each ``PVC`` is determined by dividing the capacity by the number of volumes. 
+
+           The generated claims have pod selectors so that claims are only made for volumes attached to node running the pod.
+
+           If the number of volumes exceeds the numnber of persistent volumes available on the cluster, ``MinIO`` hangs until the number of persistent volumes are available.
+  
+         - :mc:`~kubectl minio tenant create --namespace`
+
+           Each MinIO tenant requires its own ``namespace``.
+
+           When not specified, the MinIO Operator defaults to use ``minio``.
+           To specify a different namespace, use the :mc:`~kubectl minio tenant create --namespace` flag.
+
+           The namespace must already exist in the Kubernetes cluster.
+           Run ``kubectl create ns <new_namespace>`` to add one.
+
+         For example, the following command creates a new tenant with 16 volumes, total capacity of 16 Tebibytes, using the latest MinIO image, on four servers, in the ``minio`` namespace.
+
+         .. code-block:: shell
+            :class: copyable
+
+            kubectl minio tenant create minio_tenant --capacity 16Ti --servers 4 --volumes 16 --namespace minio
+
+      2) Add any optional configurations
+
+         You can further customize your tenant while executing the command by including any or all of the following *optional* flags.
+
+         - :mc:`~kubectl minio tenant create --image`
+           
+           Customize the ``minio`` image to use.
+  
+           By default, the Operator uses the latest ``minio`` release image when creating the tenant.
```

Comment:
Does it? I thought each Operator version had a static release [based on the Operator release](https://github.com/minio/operator/blob/master/pkg/apis/minio.min.io/v2/constants.go#L91). They can specify `minio/minio:latest` to use latest stable.

Reply:
Good catch.